.travis.yml

```diff
@@ -1,16 +1,25 @@
 language: dart
 
 sudo: false
+xvfb: false
 
 dart:
+  - "2.0.0"
   - stable
-  - "1.24.3"
   - dev
 
 cache:
   directories:
     - $HOME/.pub-cache
+    - .dart_tool/pub
+    - .pub/
 
-script:
-  - pub run bin/check_formatting.dart
-  - pub run test
```

Comment:
I think we need to put the tests back.

Reply:
Wait... The job appears to be running tests? How is it doing that?

Reply:
[Travis CI docs](https://docs.travis-ci.com/user/languages/dart/):
> If your package depends on the test package, pub run test will be run by default.

Reply:
If I add tests back in similar to check_formatting, then it would only run it for the first version of dart listed in the config. So I would have to explicitly state which versions to run it for.

Though it would still run the tests again, so I think it's fine as it is.

Reply:
Ah, so it's built-in to the dart job. Good deal.